public/pages/utils/helpers.ts

```diff
@@ -160,12 +160,12 @@ export const constructHrefWithDataSourceId = (
 
   // Set up base parameters for '/detectors'
   if (basePath.startsWith('/detectors')) {
-    url.set('from', '0');
-    url.set('size', '20');
-    url.set('search', '');
-    url.set('indices', '');
-    url.set('sortField', 'name');
-    url.set('sortDirection', 'asc');
+    url.set(DETECTORS_QUERY_PARAMS.FROM, '0');
+    url.set(DETECTORS_QUERY_PARAMS.SIZE, '20');
+    url.set(DETECTORS_QUERY_PARAMS.SEARCH, '');
+    url.set(DETECTORS_QUERY_PARAMS.INDICES, '');
+    url.set(DETECTORS_QUERY_PARAMS.SORT_FIELD, DETECTORS_QUERY_PARAMS.NAME);
+    url.set(DETECTORS_QUERY_PARAMS.SORT_DIRECTION, SORT_DIRECTION.ASC);
```

Comment:
You can populate all of the values using `DEFAULT_QUERY_PARAMS in the same constants file.

Reply:
sure let me update it